affectedmoduledetector/src/test/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorIntegrationTest.kt

```diff
@@ -43,17 +43,18 @@ class AffectedModuleDetectorIntegrationTest {
     @Test
     fun `GIVEN multiple project WHEN plugin is applied THEN tasks has dependencies`() {
         // GIVEN
+        tmpFolder.setupAndroidSdkLocation()
```

Comment:
I am assuming that you all have an environment variable that points to your Android SDK location...... well I didn't :stuck_out_tongue_closed_eyes: . I don't think we should assume this in general hence this workaround.

Reply:
I slightly disagree here. We are working on an android project (AMD does kind of expect a proper android setup) and so one would assume that we've got an environment variable setup to point correctly

Actually creating a file on the users seems odd but I guess its in a temporary directory so that should be fine?

Reply:
> AMD does kind of expect a proper android setup

This I agree with 💯.

> and so one would assume that we've got an environment variable setup to point correctly

Well.... I'm not sure that gets set up by default and it is manual setup that will be needed to create this environment property. At least that's the conclusion I came to months ago when o was working on this on Windows. For example simply by installing the Android SDK I didn't have this environment property available. It might work in AS because theat sets up its own if I recall. So whilst I can agree that we would assume the Android SDK is available.... it doesn't _necessarily_ have to be exposed via an environment variable.

I will try to set up a brand new project on Windows and Linux and see if they expose the property if you want?

> Actually creating a file on the users seems odd but I guess its in a temporary directory so that should be fine? 

Yeah I thought this was the _least_ amount of work since it is transparent to users and it cleans up after itself. Hopefully that's cool 👍.